### ISSUE TITLE
Can't use Quick Reply, Getting ''Connection error while posting.'' error

### ISSUE THREAD

**Issue by DesertStars** [ccd0/4chan-x#2474]
**Steps to reproduce:**

Try to post in 4chan - not 4channel - via Quick Reply.

**Expected Result:**

Succeeded at posting.

**Current Result:**

Get ''Connection error while posting.'' error. The more detailed error Mozilla gives is:

''Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://sys.4chan.org/h/post. (Reason: CORS request did not succeed).''

**More information:**

Browser: Mozilla Firefox 70.0.1 (64-bit)

Both Violentmonkey and 4chanX is up to date.

I know there's a mentioned workaround in the wiki, saying ''8 May 2017: If you are having connection errors when posting certain text, it is because of an extra captcha Cloudflare wants you to solve before allowing your post to go through to 4chan. It is not yet integrated correctly with 4chan yet (CORS headers are missing). To make your post go through, you will need to use the post form at the top of the page (click the "[Original Form]" link).''

But does this issue still persist? I thought it was resolved, am I wrong?

**Comment by DesertStars** [comment]
Already tried turning off ''Show Upload Progress'' option by the way, no luck there.

**Comment by ccd0** [comment]
What do you get when you try to post using the original post form?

**Comment by DesertStars** [comment]
In Console?

```
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “https:” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “http:” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “https:” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “http:” within script-src: ‘strict-dynamic’ specified
[Nano] Nano Defender Activated :: boards.4chan.org core.js:43:24
[Nano] Excluded :: uBO-Extra
```

And then I get the ''Post successful'' screen.

**Comment by DesertStars** [comment]
The culprit was an extension as I suspected, ''Windscribe VPN'' to be precise, going to ''Add-ons Manager'' menu and disabling the extension fixed the issue. Though  it wasn't on when I was trying to post it, otherwise I wouldn't be able to post with Original Form either as it'd just give me a warning saying your IP range is banned. 

Sorry for the inconvenience I've made, I'm closing the issue now as the problem has been solved.
